### PR TITLE
Add waitForFileComplete logics in JNI-FUSE

### LINKED ISSUE #13112: JNI-FUSE async release() cause problem in write-then-read and write-then-ls workloads
**Alluxio Version:**
2.6.0-SNAPSHOT

**Describe the bug**
JNI-FUSE release() is asynchronous which cause problem in write-then-read and write-then-ls workloads.
create a file -> write to the created file -> release() is called but doesn't finish -> getAttr() file size = 0
create a file -> write to the created file -> release() is called but doesn't finish -> open() file failed because file is not completed

**To Reproduce**
It's kind of a race condition. 
It's caught by alluxio.client.fuse.JNIFuseIntegrationTest.ls and alluxio.client.fuse.JNIFuseIntegrationTest.ddDuAndRm tests.

**Expected behavior**
write-then-read and write-then-ls workloads should not have correctness issue when using JNI-FUSE

**Urgency**
Urgent

**Additional context**

### PR DESCRIPTION
**Pull request by LuQQiu**
Fix #13112

### CODE REVIEW

File: tests/src/test/java/alluxio/client/fuse/AbstractFuseIntegrationTest.java [hunk 1]
```diff
@@ -204,6 +204,15 @@ public void cp() throws Exception {
   public void ddDuAndRm() throws Exception {
     String testFile = "/ddTestFile";
     createFileInFuse(testFile);
+    AlluxioURI testFileUri = new AlluxioURI(testFile);
+
+    CommonUtils.waitFor("File write completed", () -> {
```
Comment (apc999) [inline tests/src/test/java/alluxio/client/fuse/AbstractFuseIntegrationTest.java:209]: shall we add time out here?

Comment (apc999) [reply tests/src/test/java/alluxio/client/fuse/AbstractFuseIntegrationTest.java:209]: please also add the reason why we need to wait here (due to async release).
Let's also create a ticket on this?

**Comment by LuQQiu** [comment]
Jenkins, test this please

**Comment by LuQQiu** [comment]
Jenkins, test this please

**Comment by LuQQiu** [comment]
alluxio-bot, rerun this pleas

**Review by apc999 — APPROVED** [review]

**Comment by LuQQiu** [comment]
Error: 9.092 [ERROR] Tests run: 5, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 1.833 s <<< FAILURE! - in alluxio.master.transport.GrpcMessagingTransportTest
Error: 9.092 [ERROR] alluxio.master.transport.GrpcMessagingTransportTest.testServerClosed  Time elapsed: 1.437 s  <<< FAILURE!
java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at alluxio.master.transport.GrpcMessagingTransportTest.testServerClosed(GrpcMessagingTransportTest.java:153)

**Comment by LuQQiu** [comment]
All tests passed @apc999

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**